### PR TITLE
README: Add document format & organization conventions

### LINKED ISSUE #3: Define a directory structure for the docs
## Motivation/Why?
- We have one design doc merged under a directory.
- We are going to merge a new one (https://github.com/storj/design-docs/pull/2) under another directory.
- Both design docs are related to satellite, however:
  - The one to merge isn't currently in the _satellite_ directory as the first one.
  - The one to merge is also related to Edge services for supporting more S3 endpoints.

## What needs to be done?
- Determine a directory structure or keep the documents in the root of the repository.

## Acceptance Criteria
- In the case that we decide to have a directory structure:
  - Define if there is one level or sub-levels.
  - Define how the directory structure is created, for example per system, area, etc. The structure should be clear so future design doc writers know where to add their document (in an existing directory or creating one following the defined structure)
- Decide where the old design docs should be kept. Unblock https://github.com/storj/design-docs/issues/4
- Document in the README about the directory structure, or in the case of keeping the documents in the root, explicitly mention it.

### PR DESCRIPTION
**Pull request by ifraixedes**
Add a few conventions about the format of the documents and how to organize them in this repository.

Apply those conventions to the existing documents.

Closes #3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,31 @@
...
+  The additional files must be store in a directory located in the root of this repository with the
+  same name of the document without the `.md` extension.
+
+
+### Old blueprints
+
+Storj used to write design documents, called blueprints, in a directory of the
+[storj/storj repository](https://github.com/storj/storj).
+
+All the documents that we ported into this repository are stored in the directory `old-blueprints`
```
Comment (egonelbre) [inline README.md:30]: Do we need the separate folder? Maybe we can separate out "non-functional" designs i.e. things that have been changed?

**Review by egonelbre — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ifraixedes) [reply README.md:30]: what do you suggest? bring them to the root folder without any change? bring them to the root folder changing their name, content, etc.?

Comment (egonelbre) [reply README.md:30]: Bring them into root folder with the first commit date as the header. Any docs that are not applicable (e.g. code has been deleted), move to `archived/` folder.

Comment (egonelbre) [reply README.md:30]: And also add tags.

Comment (ifraixedes) [reply README.md:30]: The format is going to be different, we are going to create a template for our current format in #5 and I will modify the README to refer to it.

If that's fine, I don't have any inconvenient, but I'll create a separate issue and delete this sentence.

Comment (egonelbre) [reply README.md:30]: I think having a different format / template for old docs should be fine. Being able to easily find the relevant info is somewhat more important.

Comment (ifraixedes) [reply README.md]: Section deleted and issue created https://github.com/storj/design-docs/issues/8

**Review by egonelbre — APPROVED** [review]

**Pull request merged**